### PR TITLE
created payment method component

### LINKED ISSUE #168: [Feature]: SETTINGS PAGE - Implement Payment Method Component
### Description

Implement a reusable Payment Method component that receives two props: 
The title of the payment method and an image of the payment type. 
The component should include a radio input for selecting the payment method. 
The three expected payment methods are Credit Card, PayPal, and Stripe.

### Acceptance Criteria

1. The component should accept two props: `title` (string) and `image` (URL or image path).
2. The component should display the provided title and image.
3. The component should include a radio input for selecting the payment method.
4. The component should be reusable for different payment methods (Credit Card, PayPal, Stripe).
5. The component should be styled consistently with the rest of the application.
6. The component should be responsive and accessible.

Screenshot 2024-07-17 at 7 22 22 PM

### Links

https://www.figma.com/design/VEItfX6St5NSAqqNHImcxD/HNG-Boilerplate-Designs?node-id=669-13031&m=dev

### PR DESCRIPTION
**Pull request by Goldyn19**
# Description

**Closes #168 **

# Changes proposed
create payment method component

## What were you told to do?

create payment method component
## What did you do?
i created payment method component

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/Videos
Screenshot 2024-07-21 125910

Screenshot 2024-07-21 140959

### CODE REVIEW

**Review by phoenixdahdev — COMMENTED** [review]
@PrudentBird aw fa help me check this

**Comment by oreoluwa212** [comment]
Both tests failed, pull from upstream, run test, upload screenshots and that of your task

**Comment by Goldyn19** [comment]
> Both tests failed, pull from upstream, run test, upload screenshots and that of your task

i just did that

**Comment by JulianaSau** [comment]
add a screenshot of the component you created @Goldyn19

File: src/components/paymentMethod/CustomRadioButton.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+const CustomRadioButton: React.FC<CustomRadioButtonProperties> = ({
+  name,
+  value,
+  checked,
+}) => {
+  return (
+    <label className="inline-flex items-center space-x-2">
+      <input
```
Comment (JulianaSau) [inline src/components/paymentMethod/CustomRadioButton.tsx]: shadcn already has a radio group component. dont reinvent the wheel
https://ui.shadcn.com/docs/components/radio-group

**Review by JulianaSau — CHANGES_REQUESTED** [review]

**Comment by Goldyn19** [comment]
> add a screenshot of the component you created @Goldyn19

done

**Comment by Goldyn19** [comment]
i need to customize the colour of the radio button that was why i had to customize mine

File: src/components/paymentMethod/CustomRadioButton.tsx (hunk 1 above)
Comment (Goldyn19) [reply src/components/paymentMethod/CustomRadioButton.tsx]: i need to customize the color of the radio button to fit the figma design
shadcn doesnt alllow one to customize color of radio button

Comment (JulianaSau) [reply src/components/paymentMethod/CustomRadioButton.tsx]: it actually does @Goldyn19. thats what the className prop is for. and if it doesnt, you cutomize the radio component to do so. The problem with creating your own components from scratch is that they dont come with accessibility, theyre not tested etc so in the long run its more costly.

Comment (Goldyn19) [reply src/components/paymentMethod/CustomRadioButton.tsx]: i have made the necessary changes

**Pull request closed without merging**